scripts/check_doc_links.py

```diff
@@ -36,41 +41,60 @@
 RELATIVE_PATH_STR_LEN = len(RELATIVE_PATH_STR)
 INDEX_FILE_PATH = Path("docs/index.md")
 
-WHITELIST_URL_TO_CODE = {
-    "https://dl.acm.org/doi/10.1145/3212734.3212736": 302,
-    "https://s-oef.fetch.ai:443": 405,
-    "https://golang.org/dl/": 403,
-    "https://www.wiley.com/en-gb/An+Introduction+to+MultiAgent+Systems%2C+2nd+Edition-p-9781119959519": 403,
-    "https://colab.research.google.com": 403,
-    "https://github.com/fetchai/networks-stargateworld": 404,
-}
```

Comment:
No more skips for now.

Reply:
What was the purpose of this variable?

Reply:
To allow these urls to fail with that code, although there was a bug with it and we were not detecting 404 errors.